pkgs/servers/fileshare/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, lib, fetchgit, pkgconfig, git, libmicrohttpd }:
+
+stdenv.mkDerivation rec {
+  pname = "fileshare";
+  version = "0.2.4";
+
+  src = fetchgit {
+    url = https://git.tkolb.de/Public/fileshare.git;
+    rev = "v${version}";
+    sha256 = "03jrhk4vj6bc2w3lsrfjpfflb4laihysgs5i4cv097nr5cz32hyk";
+  };
+  
+  postPatch = ''
+    sed -i 's,$(shell git rev-parse --short HEAD),/${version},g' Makefile
+  '';
+
+  nativeBuildInputs = [ pkgconfig git ];
+  buildInputs = [ libmicrohttpd ];
+  
+  buildPhase = ''
+    make BUILD=release
```

Comment:
better pass this as a makeFlags else you prevent someone from using custom hooks.

Reply:
I was sure that this kind of flag existed, but was unable to find it in the documentation.
Thanks, I've done so.